cloud/pkg/admissioncontroller/config/kube.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
+)
+
+// Kube container Kubernetes related configuration
+var Kube *KubeInfo
+
+func init() {
+	Kube = NewKubeInfo()
+
+	if km, err := config.CONFIG.GetValue("controller.kube.master").ToString(); err != nil {
```

Comment:
i think it would be better if we devicecontroller.kube.master as this controller is mainly for managing devices